### PR TITLE
 Yuzu : Resized main config window 

### PR DESCRIPTION
**Pull request by SmookeFR**
Hi this is my first pull request. Sorry for multiple commits, its the sames changes but i missclick few times.
This is a porting from this pull request : https://github.com/citra-emu/citra/pull/3336/commits/8fa160395b963414fc5e32a1c3a041c4fdb93ce0

Please say me if its okay.

### CODE REVIEW

File: src/yuzu/configuration/configure.ui [hunk 1]
```diff
@@ -17,7 +18,8 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
+	  <number>0</number>
```
[the remark this answers is not in the document]
Comment (SmookeFR) [reply src/yuzu/configuration/configure.ui]: it's not intentional

[the remark this answers is not in the document]
Comment (SmookeFR) [reply src/yuzu/configuration/configure.ui]: as i said i missclick and thats why i made multiple commits

**Comment by bunnei** [comment]
I'm not sure we have any good reason to change the window size, so I'm going to close this one! Feel free to submit a followup PR or issue if you think there is a bug here that needs fixing. Thanks!

**Comment by FearlessTobi** [comment]
@bunnei I don't understand then, why it wasn't rejected in the Citra repo in the first place. At least for consistency we should imo either remove it then also from Citra or reopen it here. I also think it looks better when the window isn't so large as it is atm in yuzu. (It also is on @jroweboy's list btw)

**Comment by jroweboy** [comment]
@FearlessTobi for the second time, that was bunnei's list, i was just relinking it.

two notes:
* the default width changed in citra several by people accidentally changing the number (and no one batted an eye)
* this pr didn't do it correctly (it left the old width in there and duplicated a random field)

**Pull request closed without merging**